### ISSUE TITLE
Authority lists

### ISSUE THREAD

**Issue by hlapin** [hlapin/eRabbinica#4]
Persons   
* Nyms   
* Names   

Places   
* "Countries"
* Places

**Comment by hlapin** [comment]
for nyms (= canonical names)    
propose, for now simple table (csv) has as columns:
* ancient language form [name before hyphen] 
* transcription in English [name after hyphen]
* gender [M/F]
* language type
* ID[1]

[1]ID is necessary, b/c it was created from original name dropping non-ascii characters so may need to be altered for legibility.

**Comment by hlapin** [comment]
For persons, again a simple table (CSV)
with 
* lemma name [transcribed]
* lemma name [ancient]
* gr
* he
* en
* checkme [will need to be disambiguated as Arabic, Coptic, Persian etc.]
* nym form transcribed [should = lemma]
* nym form original [should = lemma]
* gender [M/F]
* language type
* link_to_volume

**Comment by hlapin** [comment]
List 1: persons
Here is a first cut at a list of persons. The headings that start with "person: are taken from the individual person record. The headings that start with "name:" are taken from the names (nyms) list
I plan an authority list #2 that will have the names ("nyms") only
There are at least 58 rows where the wrong information flows into the spreadsheet. Need to track that down. 

Headings
person:nym 
the name to which the person corresponds
Should match name:form
person:lemma_en
standardized spelling in transliteration
person:lemma_ancient
standardized spelling in original language
person:orthography
the forms this particular person's name(s) appear in the sources.
person:sex
M/F
person:hebrew 
person's name as rendered in source by hebrew characters 
person:greek
person's name as rendered in source by greek characters
person:checkme
names whose rendering needs to be further specified
person:id
id for the particular person
person:pointer_in_volume 
location in original publication
name:id
ID for this NAME
name:form
original heading in print lexicon
should match person:nym
name:language_type
Biblical, Hebrew-Semitic, Greek-Semitic, Latin, Greek, Arabic, Coptic, Iranian
should be reflected in person:id and person:pointer_in_volume
name:gender_type  
M/F
Should match person:sex
[preliminary persons table.xlsx](https://github.com/hlapin/eRabbinica/files/8193000/preliminary.persons.table.xlsx)

**Comment by hlapin** [comment]
List 2: nyms/names
In the spread sheet version I have highlight apparent duplicates.
Note the presence of garbage Greek.

name:id on authority list 1 should correspond to id on authority list 2

Please let me know how you want to deal with toponyms:
We have country/region names for provenance, and place names for findspots. Do you want these on the same list?

[nymsAuthorityList.xlsx](https://github.com/hlapin/eRabbinica/files/8193002/nymsAuthorityList.xlsx)
[nymsAuthorityList.csv](https://github.com/hlapin/eRabbinica/files/8193003/nymsAuthorityList.csv)

**Comment by hlapin** [comment]
List 3. "Finds" or "details" list
List 4: Provenance place names list.

**Comment by hlapin** [comment]
Summary
Here is a summary of what I have done ([@Magdalena Turska](mailto:tuurma@gmail.com), I will post a copy/summary in github issues as well as working copies of the files....)
4. Toponyms from the provenance field
I've pulled all the place names, and attached new versions to this email; the second tab of the spreadsheet has a list of (86) unique values. Where there was no provenance I assumed it was a Palestine volume and inserted "[Palestine?]" instead of an empty cell. 
The main list provides the name and the pointer in the (as an ID; the ID is less good for human readers)
Looking just at the first few unique values (2d tab) for a second we can see that there are problems created in processing ('abylonia', 'Arab. Pen.D: Pre-627 CE') but also differences created from very slight differences in the original (spacing, periods). 
I would say we want to come up with a standard name for each of these places and also a method to handle uncertainty explicitly:
'Asia?', 'Babylonia?' where uncertainty is marked by question mark
'Asia / Crete' where one of two places is possible.
3. "details_aka_findspot" This is Tal's "Find" field and it consists of different kinds of records:
Objects in their presumed origin
Incantation bowl--Nippur
Incantation bowl--Nippur--Montgomery 
Objects in their current location 
Incantation bowl--London 
Incantation bowl--Harvard--Gordon 
Objects by their conventional collection name:
Papyrus--Zenon Archive-- Kalbfleisch 
Literary references, which can be complex (multiple references, bibliography for each):
1.BJ2:520;AJ20:32 (Schalit NB 87); 2.mYoma3:10 (Kasovsky Mishnae 1141); 3. Tacitus Dio Casius GLAJJ2: 85 368
[Note: these may or may not also have a shorthand bibliography:
Montgomery,
Gordon,
also Ossuary lid-- Bethphage--Dussaud 1910 CIJ no. 1285] 
My recommendation is to create new columns for separating out the data and some standardized entries for those columns:  (object type [=ossuary, epitaph, literary ref ...]; findspot; current repository; literary reference (source); bibliography, etc. and to leave columns blank where there is no data.

PROBLEM: bibliographical references with commas in the original got chopped up into multiple columns; the alternative is to pass the contents of the original text into a single cell. My intuition was that it was easier to join stuff from different columns than to split stuff out

2. nyms/names Authority List.
Here, like the second tab on the placeNameAuthorityList, we have a list of unique values, but 
there are clearly entries that should be merged; 
and we may want to update unreadable IDs with readable ones (I stripped most special characters out of the ID thus: 
ĀbānDūχt-ĀbānDūχt has the ID: name-bnDt-d1e636299
1. preliminary persons table
Tal has done a first pass on this, and Hayim has not gotten to it yet!
